hack/templates/values/values.template

```diff
@@ -25,6 +25,9 @@ psp:
 
 upgradeCRDs: true
 
+global:
```

Comment:
why would you not put it after deployCR with the rest of nicclusterpolicy parameters.

also id drop the "global" as we didnt  add it to begin with for the top level items of the CR deployed by the chart.

Reply:
Removed the 'global' section and put it together with NicClusterPolicy values